### ISSUE TITLE
[Error] boost::filesystem::directory_iterator::construct: No such file or directory

### ISSUE THREAD

**Issue by AlbrechtL** [linuxdeploy/linuxdeploy-plugin-qt#122]
On Ubuntu 22.04 linuxdeploy-plugin-qt is throwing the following error:
`[qt/stdout] Deploying file /home/travis/build/AlbrechtL/welle.io/6.2.4/gcc_64/qml/QtCharts/designer/images/percentbarserie[qt/stderr]   what():  boost::filesystem::directory_iterator::construct: No such file or directory: "/home/travis/build/AlbrechtL/welle.io/6.2.4/gcc_64/plugins/mediaservice"`

This happens with the Qt6 libs inside the repository as well as the official Qt6 libs from qt.io. I was able to workaround that with manually creating the two following directories
```
mkdir /home/travis/build/AlbrechtL/welle.io/6.2.4/gcc_64/plugins/mediaservice
mkdir /home/travis/build/AlbrechtL/welle.io/6.2.4/gcc_64/plugins/audio
```

**Comment by bjorn** [comment]
This was already fixed by #115, but it still needs to be merged.

**Comment by TheAssassin** [comment]
#115 has been merged since.
